### DISCUSSION TITLE
New Contributor

### DISCUSSION THREAD

**Discussion by oumayma122** [cypress-io/cypress#15684] [General]
hello everyone, I'm very impressed about your project and I wanna start contributing to .. Please who can help me about the first steps to begin?

**Comment by pedrohyvo** [comment]
@oumayma122 Please see our [Contributing Guideline](https://github.com/cypress-io/cypress/blob/develop/CONTRIBUTING.md) which explains repo organization, linting, testing, and other steps. 👍
